### DISCUSSION TITLE
Request: More languages

### DISCUSSION THREAD

**Discussion by debianis** [growilabs/growi#8190] [Ideas / Requests]
Hello, is there any chance of implementing more languages ?
Maybe ure already using kind of translation files for buttons, menues etc.?

**Comment by yuki-takei** [comment]
> Hello, is there any chance of implementing more languages ?

Yes. We plan to add languages and manage with [Locize](https://locize.com/) so that communities can maintain those.

However, that has not been done yet.

**Comment by Unicom3000** [comment]
I would really like to see the possibility of translation into other languages
